lib/elasticsearch_relation.rb

```diff
@@ -20,7 +19,7 @@ def all
   end
 
   def count
-    Annotation.search(@params.merge({ size: 0 })).total
+    Annotation.search(@params).total
```

Comment:
If we have more than 10.000 the `total` will be wrong... I think you shouldn't change this line... or is there a reason I'm not seeing?